CHANGELOG.md

```diff
@@ -3,11 +3,12 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v0.1.1 - [2023-06-26]
+## v0.1.1 - [2023-06-30]
```

Comment:
reminder to update date

Reply:
thanks! Addressed those TODOs and will update the version/date accordingly. For the template update will it be a minor release (so up to 0.2.0)?

Reply:
why no 1.0 yet?

Reply:
During the first release there was some concern as to whether the pipeline was "nf-core enough" mainly things like this about it being lifted from CTAT-VIF:
https://github.com/nf-core/viralintegration/pull/52#discussion_r1084426780
https://github.com/nf-core/viralintegration/pull/52#discussion_r1084431649

There was more discussion between @Emiller88 and the core team on slack I believe to release it as 0.1.0 instead of 1.0.0 until those issues are addressed. They're in the works, but wanted to get this one out there for the README fix and template updates.